.circleci/config.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       - setup
       - run: pip install prospector>=1.2 GitPython~=2.1
       - run: pip install .
-      - run: c=`python ci/evaluate_docs.py`; if [ -z $c ]; then echo "No .py files to lint"; else echo $c | xargs prospector; fi
+      - run: prospector . 
```

Comment:
I think you can just run `prospector` without specifying a directory. It's how I've been running it anyway.